src/Components/Projects/index.js

```diff
@@ -27,6 +27,24 @@ const Projects = props => {
 
     item.display = true;
 
+    const LabelsCollection = new Collection(
+      props.firebase,
+      `/projects/${ docRef.id }/labels`
+    );
+
```

Comment:
move this to Projects, below `collection` L:17, so we have one place to have similar assignments

Reply:
I don't think this will work as Projects doesn't have a docRef.id or projectId - those values are only assigned within the createProject function...

Reply:
oh snap! my bad 😓